src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -38,4 +38,30 @@ public static function castDateTime(\DateTimeInterface $d, array $a, Stub $stub,
 
         return $a;
     }
+
+    public static function castInterval(\DateInterval $interval, array $a, Stub $stub, $isNested, $filter)
+    {
+        $i = array(Caster::PREFIX_VIRTUAL.'interval' => self::formatInterval($interval));
```

Comment:
what about using a ConstStub here that gives the number of seconds on hover (html mode):
`new ConstStub(self::formatInterval($interval), $numberOfSeconds);`